### DISCUSSION TITLE
Discussion page : cannot "View more" ?

### DISCUSSION THREAD

**Discussion by pcar75** [jamulussoftware/jamulus#2815] [Bug Investigation]
Bonjour,
there seems to be a problem with the 'View more' action in a Discussion page :
[https://github.com/jamulussoftware/jamulus/discussions/2807](url)
One of my comment says 6 replies and displays 5, When I click the 'View more' action to see the 6th reply, all replies disappear and the main comment displays 0 reply.

 ---------Before---------------
![GitHub_Jamulus Discussion_01](https://user-images.githubusercontent.com/64435162/187092785-3e053809-6a13-41ce-bb89-47bdefdaad14.PNG)

----------After-----------------
![GitHub_Jamulus Discussion_02](https://user-images.githubusercontent.com/64435162/187092787-2585eca6-5d2f-44ca-bb25-d2bfb6d04188.PNG)

?!?! I never experienced this before and I remember reading Topics having comments + View more with a lot more and longer replies,

**Comment by ann0see** [comment]
Probably you should post that here: https://github.com/community/community/discussions/categories/discussions

**Reply by pcar75** [reply]
Thanks .  Done . 
[https://github.com/orgs/community/discussions/31174](url)

**Reply by ann0see** [reply]
Great!

Your link seems broken. 
This one works https://github.com/community/community/discussions/31174

**Comment by ann0see** [comment]
Locked as discussion should continue in the other thread linked here: https://github.com/jamulussoftware/jamulus/discussions/2815#discussioncomment-3497624
